src/include/daos_array.h

```diff
@@ -33,17 +33,15 @@ typedef struct {
 	daos_size_t		arr_nr;
 	/** Array of ranges; each range defines a starting index and length. */
 	daos_range_t	       *arr_rgs;
-	/*
-	 * On read only: return the number of records that are short fetched
-	 * from the largest dkey(s). This helps for checking for short reads. If
-	 * this values is not zero, then a short read is possible and should be
-	 * checked with daos_array_get_size() compared with the indexes being
-	 * read.
+	/** (on read only) number of records that are short fetched from the largest dkey(s).
+	 * Helps for checking short reads. If nonzero, a short read is possible and should be
+	 * checked with daos_array_get_size() compared with the indexes being read.
 	 */
 	daos_size_t		arr_nr_short_read;
 	daos_size_t		arr_nr_read;
```

Comment:
we can add a comment on this that here:
(on read only) contains the number of records that were actually read from the array.